addon/styles/addon.css

```diff
@@ -44,8 +43,18 @@
   padding-bottom: 25px;
 }
 
+.-horizontal .dragSortItem.-placeholderAbove {
```

Comment:
You should never be using modifier classes (that start with `-`) individually.

They must always come paired with a semantic class like this: `.dragSortList.-horizontal`.